prog/postgres/postgres_resource_nexus.rb

```diff
@@ -40,13 +41,17 @@ def self.assemble(project_id:, location:, name:, target_vm_size:, target_storage
       postgres_resource = PostgresResource.create_with_id(
         project_id: project_id, location: location, name: name,
         target_vm_size: target_vm_size, target_storage_size_gib: target_storage_size_gib,
-        superuser_password: superuser_password, parent_id: parent_id,
+        superuser_password: superuser_password, ha_type: ha_type, parent_id: parent_id,
         restore_target: restore_target
       )
       postgres_resource.associate_with_project(project)
 
       PostgresFirewallRule.create_with_id(postgres_resource_id: postgres_resource.id, cidr: "0.0.0.0/0")
-      Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: timeline_id, timeline_access: timeline_access, representative: true)
+
+      Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: timeline_id, timeline_access: timeline_access, representative: true, synchronization_status: "ready")
+      postgres_resource.required_standby_count.times do
+        Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: timeline_id, timeline_access: "fetch", representative: false, synchronization_status: "catching_up")
+      end
```

Comment:
setting `synchronization_status` in the `.assemble` feels... out of place. Shouldn't it be `catching_up` for `not representative` servers by default, and `ready` for representative servers. Should I be able to create a `not representative` server in `ready` state?

Reply:
Makes sense. Changed it.